modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -37,6 +37,29 @@ ifdef::ovn[]
 - A protocol that is one of the following protocols: TCP, UDP, and SCTP
 endif::ovn[]
 
+[IMPORTANT]
+====
+You must allow access to the {product-title} API servers or connections to the API endpoints are blocked. For example, you must define egress firewall rules similar to the following:
+
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: allow-from-api-endpoints
+spec:
+  egress:
+  - to:
+      cidrSelector: 74.6.0.0/16
+    type: Allow
+  - to:
+      cidrSelector: 10.0.0.0/16
```

Comment:
This API server range might be different for customer clusters.  We can give them the example and how to get this like the description in the bug. Thanks!
oc get ep -n default